src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+
+import { ShowcaseProperty } from '~/app/shared/showcase-properties/showcase-property';
+
```

Comment:
I get a build error here on `require``

I can fix it by adding:

```suggestion
declare var require: any;
```

Reply:
Done :)